### DISCUSSION TITLE
support for .safetensors ?

### DISCUSSION THREAD

**Discussion by DiegoRRR** [hlky/stable-diffusion-webui#1715] [Feature suggestions]
Hey.
Is it planed to support .safetensors model files ?
More and more models will use this format only now, and .ckpt should disappear, for .safetensors can't contain malware.

**Comment by bluedevil678** [comment]
I have to second this, progress is clear here and support for this would be game changing!

**Reply by arkique** [reply]
safetensors support would be amazing :D
